### DISCUSSION TITLE
Not showing commits in the activity table

### DISCUSSION THREAD

**Discussion by ProfessorMR** [community/community#76944] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello
There are some times that when I commit, my commits are not displayed in the table, even though I have them in the commits section of the repositories.
This issue happens when I'm working on an external repository (I mean it's not mine and I forked it), but when I'm working on my own repository, everything is fine, even I enabled the settings to show private commits in my profile.

**Comment by guspan-tanadi** [comment]
Hi, here is by citing GitHub Docs [Missing contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commit-was-made-in-a-fork).

> Commits made in a fork will not count toward your contributions.

Unless create a pull request so the commit can be merged to repository that has been fork before or other option by detach the fork accordingly.

---

Seem related to fork on GitHub Desktop there are settings for [Fork Behavior](https://docs.github.com/en/desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop#managing-fork-behavior) which provide option for what the fork will be in used.

**Comment by mateusloubach** [comment]
I have the same issue, but I did not use Github Desktop or any other means. just same as always and it just stopped working all of the sudden..

**Comment by faridnec** [comment]
it might be due to the repository owner's settings. Some repositories don't count contributions from forked repositories by default. Maybe you can confirm it to the owner of the repository regarding their contribution settings or contact the owner to confirm their policies.

**Reply by mateusloubach** [reply]
it`s actually my repo and it's the same repo I've been commiting to for the past 10 days. the other days it worked fine, but all of the sudden, github has stopped recording my progress with any repo I commit to.. I did not alter any settings.
